model/infinispan/src/main/java/org/keycloak/cluster/infinispan/LockEntryPredicate.java

```diff
@@ -77,7 +77,7 @@ public LockEntryPredicate readObject(ObjectInput input) throws IOException, Clas
 
         public LockEntryPredicate readObjectVersion1(ObjectInput input) throws IOException, ClassNotFoundException {
             return new LockEntryPredicate(
-                    KeycloakMarshallUtil.readCollection(input, KeycloakMarshallUtil.STRING_EXT, size -> new ConcurrentHashSet<>())
+                    KeycloakMarshallUtil.readCollection(input, KeycloakMarshallUtil.STRING_EXT, size -> ConcurrentHashMap.newKeySet())
```

Comment:
```suggestion
                    KeycloakMarshallUtil.readCollection(input, KeycloakMarshallUtil.STRING_EXT, ConcurrentHashMap::newKeySet)
```
The lamba can be avoided.

Reply:
@martin-kanis Corrected per the suggestion in the latest version (please double-check). Thanks!